test/just.test.js

```diff
@@ -0,0 +1,30 @@
+const just = require("../src/just")
```

Comment:
because of the `exports` field in package.json (supported node 12+), we can reference the implementations like this:

```js
require("monio/just")
```